tensorflow/examples/image_retraining/retrain.py

```diff
@@ -268,7 +268,7 @@ def get_image_path(image_lists, label_name, index, image_dir, category):
     tf.logging.fatal('Category does not exist %s.', category)
   category_list = label_lists[category]
   if not category_list:
-    tf.logging.fatal('Category has no images - %s.', category)
+    tf.logging.fatal('Category %s has no images - %s.', label_name, category)
```

Comment:
`label_name` is not the name of the category. `category` is. So this change makes things confusing.